src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -223,6 +227,38 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
     'health_care_region_health_services_page.drupal.liquid',
   );
 
+  // Create the patient and family services page
+  const fsEntityUrl = createEntityUrlObj(drupalPagePath);
+  const fsObj = Object.assign(
+    {
+      careCoordinatorPatientFamilyServices:
+        page.careCoordinatorPatientFamilyServices,
+    },
+    {
+      socialProgramsPatientFamilyServices:
+        page.socialProgramsPatientFamilyServices,
+    },
+    {
+      healthWellnessPatientFamilyServices:
+        page.healthWellnessPatientFamilyServices,
+    },
+    { fieldPatientFamilyServicesIn: page.fieldPatientFamilyServicesIn },
+    { facilitySidebar: sidebar },
+    { entityUrl: fsEntityUrl },
+    { title: page.title },
```

Comment:
i think we need to add `{ alert: page.alert }` to get the global alerts

Reply:
Added!